### PR TITLE
#5 trackit user rest controller

### PR DESCRIPTION
**Pull request by florin-grozea-old**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -27,6 +27,15 @@
 	<dependencies>
 		<dependency>
 			<groupId>org.springframework.boot</groupId>
+			<artifactId>spring-boot-starter-web</artifactId>
+		</dependency>
+		<dependency>
```
Comment (florin-grozea-old) [inline pom.xml:32]: Remove this dependency.

File: src/main/java/io/trackit/dto/CreateTrackitUserDto.java [hunk 2]
```diff
@@ -0,0 +1,24 @@
+package io.trackit.dto;
+
+
+//create controller.
```
Comment (florin-grozea-old) [inline src/main/java/io/trackit/dto/CreateTrackitUserDto.java]: Remove this comment.

File: src/test/java/io/trackit/controller/UrlConfigControllerIntegrationTests.java [hunk 3]
```diff
@@ -0,0 +1,62 @@
...
+    public void setUp() {
+        trackitUserRepository.deleteAll();
+        trackitUser = new TrackitUserCreator("userName", trackitUserRepository).create();
+    }
+
+    @Test
+    public void createUrlConfig() throws Exception {
+        CreateUrlConfigDto createUrlConfigDto = createCreateUrlConfigDto();
+
+        ResponseEntity responseEntity = restTemplate.postForEntity("http://localhost:8080/urlConfig", createUrlConfigDto, null);
```
Comment (flgor) [inline src/test/java/io/trackit/controller/UrlConfigControllerIntegrationTests.java]: extract this into a helper class to be used by all tests.

File: src/test/java/io/trackit/repository/UrlConfigRepositoryIntegrationTests.java [hunk 4]
```diff
@@ -22,15 +24,23 @@
     public void setUp() throws Exception {
         urlConfigRepository.deleteAll();
         super.setUp();
-        new UrlConfigCreator(trackitUser, TEST_URL, urlConfigRepository).create();
     }
```
Comment (florin-grozea-old) [inline src/test/java/io/trackit/repository/UrlConfigRepositoryIntegrationTests.java:27]: remove end line.

File: pom.xml (hunk 1 above)
Comment (florin-grozea-old) [reply pom.xml:32]: Done.

File: src/main/java/io/trackit/dto/CreateTrackitUserDto.java (hunk 2 above)
Comment (florin-grozea-old) [reply src/main/java/io/trackit/dto/CreateTrackitUserDto.java]: Done.

File: src/test/java/io/trackit/repository/UrlConfigRepositoryIntegrationTests.java (hunk 4 above)
Comment (florin-grozea-old) [reply src/test/java/io/trackit/repository/UrlConfigRepositoryIntegrationTests.java:27]: No end of line found. Fixed typo.

**Pull request merged**